commons-app/pom.xml

```diff
@@ -34,6 +30,15 @@
 			<name>SpringSource Enterprise Bundle Repository - SpringSource Bundle Releases</name>
 			<url>http://repository.springsource.com/maven/bundles/release</url>
 		</repository>
+
+		<repository>
+			<id>3rdParty</id>
```

Comment:
We had to solve similar issue in one plugin, also due to license restrictions. Please check this pom https://github.com/UnifiedViews/Plugins/blob/master/e-relationalFromSql/pom.xml
Also here is info how to cope with build when you want to bundle such artifact: https://github.com/UnifiedViews/Plugins/blob/master/e-relationalFromSql/BUILD.md